Feat/top investors section

respp opened the pull request:
## [video here](https://www.loom.com/share/a6dd4b26f7b1468788ab2040548694a5) 🎯


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

- **New Features**
	- Introduced a new Investors Dashboard in the community section that displays detailed investor profiles and impact metrics.
	- Added interactive investor listings with animated cards, grid layouts, and intuitive filtering and sorting options.
	- Launched a call-to-action section encouraging users to join as top investors.
	- Improved FAQ accessibility with enhanced keyboard navigation for a better user experience.
	- Implemented a responsive layout for displaying investor data, including a no-results message for user feedback.
	- Added animation effects to various components for enhanced visual appeal.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

AndlerRL commented on apps/web/components/sections/community/investor-grid.tsx:
```diff
@@ -0,0 +1,42 @@
+		<AnimatePresence mode="wait">
+			<motion.div
+				key={`${sortBy}-${category}`}
+				className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6"
```
Here should be the place! I noticed the cards are not with the same height. Setting at least the className `auto-rows-fr` should fix the issue :wink:

AndlerRL commented on apps/web/lib/constants/community-data/hero-data.ts:
Can we move this to the `apps/web/lib/mock-data` directory? :pray:

AndlerRL commented on apps/web/lib/constants/community-data/top-investors-data.ts:
Same as previous feedback to move to another directory.

AndlerRL commented on apps/web/components/sections/community/investor-card.tsx:
```diff
@@ -0,0 +1,126 @@
+
+export function InvestorCard({ investor }: { investor: Investor }) {
+	return (
+		<motion.div variants={fadeInUp}>
```
Maybe here you will also need to set `h-full` so all card with a few of content can use the whole height.

AndlerRL commented on apps/web/components/pages/Investors.tsx:
Can we rename this file to `kindlers.tsx`? :pray: To match with the labeling and not adding more abstraction.

AndlerRL reviewed (changes requested):
Changes looks very good! Good hide/show animation :+1: however, I would like to have this section to have the actual labeling for investors. These are `Kindlers`, the contributors (investors) of projects (Kinders).

Please proceed to make the labeling changes. Additionally, I found that not all Kindlers cards are having the same height on each row, I made some notes about it. After that and a few observations, it will be good to go!

PD: Check the build, it is failing :eyes:

respp commented:
Hello @Bran18,

I've updated the labeling to use "Kindlers" instead of "Investors" and applied layout fixes to ensure all cards have the same height. Also, I moved the mock data to `apps/web/lib/mock-data/` as suggested.

Additionally, I fixed the build error. Everything should be good to go now! 🚀  

Thanks for the feedback!  

C.C. @AndlerRL

Bran18 reviewed (approved):
# ✅ PR Approved – Great Job!  

Congratulations on successfully completing this issue! You did an **amazing job**, and we truly appreciate your contribution to KindFi. 🚀  

Your work has been reviewed and merged—thank you for your dedication and attention to detail.  

We **can’t wait to see your future contributions** and to continue building together. Keep up the great work! 💪  

Best,  
**The KindFi Team**
